GitExtensions/Program.cs

```diff
@@ -110,7 +110,7 @@ private static void RunApplication()
                 {
                     typeof(GitUI.GitExtensionsForm).Assembly,
                     typeof(GitCommands.GitModule).Assembly,
-                    typeof(ResourceManager.GitPluginBase).Assembly
+                    typeof(ResourceManager.LinkFactory).Assembly
```

Comment:
This change is essentially a no-op - we are registering an assembly from which MEF is loading exports.
Does LinkFactory need to be marked as Export?

Reply:
> This change is essentially a no-op - we are registering an assembly from which MEF is loading exports.

Yes, that's why "wild guess".

> Does LinkFactory need to be marked as Export?

It *is* exported.